.travis.yml

```diff
@@ -39,7 +32,6 @@ script:
         --migration-path ./django_netjsonconfig/migrations/ \
         --skip-checkmakemigrations
   - |
-    if [[ $TRAVIS_PYTHON_VERSION == 3.5 ]]; then
```

Comment:
have you removed the end delimitator of the `if`? You can also unindent the subsequent lines according to the if removal